src/index.html

```diff
@@ -5,8 +5,434 @@
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__link">
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="products">
+        <div class="card card--margin" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt=" photo of APPLE A1419 iMac 27 Retina 5K Monoblock"
+            class="card__photo"
+          >
+
+          <h2 class="card__info">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="card__rating">
+            <div class="stars stars--number_4">
+              <div class="stars__img"></div>
+              <div class="stars__img"></div>
+              <div class="stars__img"></div>
+              <div class="stars__img"></div>
+              <div class="stars__img"></div>
+            </div>
+
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">
+              Price:
+            </p>
+
+            <p class="card__price-number">
+              $2,199
+            </p>
+          </div>
+
+          <a href="#"
+            class="card__buy-button"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card card--margin">
```

Comment:
You don't need a modifier for the margin. Just do it as an element
```suggestion
        <div class="main__card card">
```